src/reducers/polls.js

```diff
@@ -0,0 +1,24 @@
+import { RECEIVE_POLLS, ADD_POLL } from '../actions/polls';
+import { ADD_ANSWER } from '../actions/answers';
+
+export default function polls(state = {}, action) {
+  switch (action.type) {
+    case RECEIVE_POLLS:
+      return { ...state, ...action.polls };
+    case ADD_POLL:
+      return { ...state, [action.poll.id]: action.poll };
+    case ADD_ANSWER:
+      const { answer, id, authedUser } = action;
+      const poll = state[id];
+      const votesKey = answer + 'Votes';
```

Comment:
thats a little bit strange here. in general the way its being used in the project can be improved.

As it is now in case you later on wanna change the application to have 5 or 6 answers is not possible I believe. you will have to go to several files and change the `["aVotes", "bVotes", "cVotes", "dVotes",]` etc etc

Reply:
Yes, to add more answers would be a little weird. I am going to think of a better API to handle this for the future.